nautobot/extras/filters/__init__.py

```diff
@@ -1062,6 +1062,9 @@ class RoleFilterSet(NautobotFilterSet):
         conjoined=False,
     )
 
+    def filter_queryset(self, queryset):
+        return super().filter_queryset(queryset).distinct()
```

Comment:
Is this something that should be implemented directly on `NautobotFilterSet` rather than on `RoleFilterSet` specifically? Curious why it's needed here only - is it because of `conjoined=False` on the content_types filter? Do we have any other `conjoned=False` filters elsewhere that would merit a similar fix?

Reply:
Move logic to `NautobotFilterSet`.  This should fix for all other places where `conjoined=False`